src/ai/defaultai.cc

```diff
@@ -1917,7 +1917,7 @@ void DefaultAI::update_buildable_field(BuildableField& field) {
 	if (flags_count > 0) {
 		field.average_flag_dist_to_wh /= flags_count;
 	}
-	// printf("flags count: %2d, avg: %3d\n", flags_count, field.average_flag_dist_to_wh);
+	// log_dbg("flags count: %2d, avg: %3d\n", flags_count, field.average_flag_dist_to_wh);
```

Comment:
if we make this only printed in verbose we might easily uncomment this to make the change sensible.